packages/core/src/i18n.ts

```diff
@@ -173,31 +173,25 @@ export class I18n extends EventEmitter<Events> {
   }
 
   /**
-   * @param locales one locale or array of locales.
-   *   If array of locales is passed they would be used as fallback
-   *   locales for date and number formatting
+   * @param locale initial active locale.
+   * @param locales list of alternative locales (BCP 47 language tags) which are used for number and date formatting.
    * @param messages compiled message catalog
-   * @param notify Should emit `change` event for all subscribers.
-   *    This is useful for integration with frameworks as NextJS to avoid race-conditions during initialization.
    */
-  loadAndActivate(
-    locales: Locale | Locales,
-    messages: Messages,
-    notify = true
-  ) {
-    if (Array.isArray(locales)) {
-      this._locale = locales[0]
-      this._locales = locales
-    } else {
-      this._locale = locales
-      this._locales = null
-    }
+  loadAndActivate({
+    locale,
+    locales,
+    messages,
+  }: {
+    locale: Locale
+    locales?: Locales
+    messages: Messages
+  }) {
+    this._locale = locale
+    this._locales = locales || undefined
 
     this._messages[this._locale] = messages
 
-    if (notify) {
-      this.emit("change")
-    }
+    this.emit("change")
```

Comment:
just curious, what is the rationale here that always notify ? since previously, people may assume that `change` is only notified if locale changed, this might break that assumption

Reply:
> since previously, people may assume that change is only notified if locale changed

`change` was (and still is) previously emitted when calling `activate` or `load`, so this is in line with the current behavior